spyderlib/plugins/__init__.py

```diff
@@ -124,6 +126,14 @@ def __init__(self, *args, **kwargs):
         if sys.platform == 'darwin':
             self.setStyleSheet(self.DARWIN_STYLE)
 
+        # Needed for the installation of the event filter
+        self.title = args[0]
+        self.main = args[1]

```

Comment:
I see. Then you need to redefine the __init__ method of `SpyderDockWidget` to accept the right args, i.e. something like

```python
def __init__(self, title, main, **kwargs):
    super(SpyderDockWidget, self).__init__(**kwargs)
    self.title = title
    self.main = main
    ...
```